packages/react-components/src/components/styles/ReactionButton.styles.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(reaction) */
+import React from 'react';
+
+/* @conditional-compile-remove(reaction) */
+/**
+ * @param backgroundImage - the uri for the reaction emoji resource
+ * @param animationPLayState - the value is either 'running' or 'paused' based on the mouse hover event
+ *
+ * @private
+ */
+export const emojiStyle = (backgroundImage?: string, animationPLayState?: string): React.CSSProperties => {
+  return {
```

Comment:
nit: if these are living inside a default fluent button might be good to look at mapping this out to one of their style sets that correspond to the button. This is how we do styles throughout the repo and we should try to match here as well

Reply:
I would use the same style object you built here but just use the `IButtonStyles` type from Fluent, would also look something like this 
![image](https://github.com/Azure/communication-ui-library/assets/94866715/31cae13c-b39a-48ad-a834-2f7b3902db6c)
This is from the `EndCallButton` your styles would go in the `root` selector section you can see in the image

Reply:
We tried to have the div elements of the button as Stack and childs as Stack.Item and have the styles as FluentStyles. But the resources did not render as expected. So reverting to the initial implementation.

Reply:
Noting this as an investigation point for us to take a look in future.